src/ManifestSchema/ManifestSchema.h

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 #pragma once
 
+// Do not use 0 as this will cause load schema to fail.

```

Comment:
Nit: I think it may be good to say why it would fail. Like "0 is used to represent when there is no resource". Maybe even have a `#define` for that special value.